prepare-chroot-base

```diff
@@ -93,5 +104,11 @@ if ! [ -f "${INSTALLDIR}/tmp/.prepared_base" ]; then
         cp -a /dev/$f $INSTALLDIR/dev/
     done
 
+    # TODO: check if there is a way to force baseurl to override metalink
```

Comment:
Well, you can create own metalink pointing to your local mirror... It's just an xml file. But probably not worth it.

Reply:
Indeed, I also though about a .repo file directly but it means to create for each FC a repo/xml file...

Reply:
We could imagine to create in the future something doing it automatically.